libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,375 @@
+  "ReportType": "originating-device-export",
+  "ReportingDevice": [
+    {
+      "@type": "CVR.ReportingDevice",
+      "SerialNumber": "scanner-1"
+    }
+  ],
+  "ReportGeneratingDeviceIds": ["scanner-1"],
+  "Party": [
+    {
+      "@type": "CVR.Party",
+      "@id": "0",
+      "Name": "Mammal Party",
+      "Abbreviation": "Ma"
+    },
+    {
+      "@type": "CVR.Party",
+      "@id": "1",
+      "Name": "Fish Party",
+      "Abbreviation": "F"
+    }
+  ],
+  "Election": [
+    {
+      "@type": "CVR.Election",
+      "@id": "some-id",
+      "Candidate": [
+        {
+          "@type": "CVR.Candidate",
+          "@id": "seahorse",
+          "Name": "Seahorse",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "salmon",
+          "Name": "Salmon",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "manta-ray",
+          "Name": "Manta Ray",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "pufferfish",
+          "Name": "Pufferfish",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "rockfish",
+          "Name": "Rockfish",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "triggerfish",
+          "Name": "Triggerfish",
+          "PartyId": "1"
+        }
+      ],
+      "Contest": [
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "best-animal-fish",
+          "Name": "Best Animal",
+          "NumberElected": 1,
+          "VotesAllowed": 1,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["seahorse"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["salmon"]
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "aquarium-council-fish",
+          "Name": "Zoo Council",
+          "NumberElected": 2,
+          "VotesAllowed": 2,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["manta-ray"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["pufferfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            }
```

Comment:
Our system infers write in positions with `allowWriteIns` but here we explicitly list them, because there's not a similar option in CDF. Same sort of thing in Ballot Definition CDF if I remember correctly.